test/functional/user_mail_notification_test.rb

```diff
@@ -0,0 +1,32 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+
```

Comment:
:heart:

I'd also prefer to write just one test per PR, but if we've got updated test data and more than one command, then it will be nice to cover them as well.

Reply:
I added more tests to all of the commands. You are definitely right there should be at least one test to each command.